src/adapters/mendi-finance/products/supply/mendiFinanceSupplyAdapter.ts

```diff
@@ -84,7 +84,7 @@ export class MendiFinanceSupplyAdapter
    * Implementation Steps:
    * 1. Implement logic for handling predefined actions (e.g., Supply, Withdraw). Consider the examples provided as a starting point.
    * 2. For new actions (e.g., Stake, Flash Loan), first extend the `WriteActions` object to include these new actions.
-   * 3. Update `GetTransactionParamsInput` type to include the parameters required for any new actions you add.
+   * 3. Export WriteActionInputs, GetTransactionParamsSchema and GetTransactionParams from this file.
```

Comment:
Does this need to be updated in the templates too?

Reply:
if so maybe we can make it one string thats reused in all templates e.g. const writeAdapterComment = '.....'

Reply:
It's been updated there too.

Reply:
What we need is now a better explanation of what it is.

But I'm going to create a ticket to automate the creation of `GetTransactionParamsSchema` object and type, then we can add the explanation there, which will be much simpler.